projects/elife.yaml

```diff
@@ -370,9 +370,11 @@ api-gateway:
                 gcslogging:
                     bucket: end2end-elife-fastly
                     path: api-gateway--test/
-                    period: 1800
+                    period: 600
                 healthcheck:
                     path: /ping-fastly
+                    check-interval: 30000
+                    timeout: 10000
```

Comment:
10 seconds seems a bit long, looks like it was 0.5 before?

Reply:
can be tuned, this is a large precautionary value that makes the POPs from around the world happy